repos/spack_repo/builtin/packages/hdf_eos2/package.py

```diff
@@ -17,66 +17,76 @@ class HdfEos2(AutotoolsPackage):
     """
 
     homepage = "https://hdfeos.org"
-    # Starting with @3, download requires authentication.  So reverting
-    # to a manual download
-    url = "file://{0}/hdf-eos2-3.0-src.tar.gz".format(os.getcwd())
+    url = "https://git.earthdata.nasa.gov/projects/DAS/repos/hdfeos"
+    local_url = "file://{0}/hdf-eos2-{1}-src.tar.gz"
```

Comment:
See my comment below. Can we remove the local_url and old_url stuff, even if that means removing deprecated versions v2 altogether?

Reply:
@climbfuji I can remove `can_auto_download`, `local_url`, and `old_url`. All versions `3.0`, `2.20v1.00`, `2.19v1.00`, `2.19b` have archive files that can be downloaded using the https raw file interface.  The only reason I deprecated `2.19b` is because it's binaries and I don't think this recipe is set up to install binaries. Plus `2.19b` fails when it tries to apply `hdf-eos2.configure.patch` because there's no `configure` file. I added `2.19v1.00` so that this version can still be installed, but from source.